package.json

```diff
@@ -11,7 +11,8 @@
     "build": "tsc",
     "start": "tsx index.ts",
     "dev": "tsx watch index.ts",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "install:challenges": "tsc install.ts && node install.js"
```

Comment:
You could just use `tsx install.ts` unless I am missing something. Seems to work for me. This also means there is no `install.js` created so that can be removed for the .gitignore.

Reply:
tsx is basically a replacement for ts-node that I wanted to try. We may find that ts-node works better if we encounter issues.

Reply:
ah yes thank you for calling attention to `tsx` and explaining :heart: 

:sweat_smile: apologies on my noob understanding of typescript config and execution. Been using the language for a while with react, but still new to config setup executing scripts written with ts